ResearchKit/Common/ORKAnswerFormat.m

```
@@ -1927,10 +1928,21 @@ - (ORKQuestionType)questionType {
     return ORKQuestionTypeText;
 }
 
+- (void)validateParameters {
+    [super validateParameters];
+    
+    if ( (!self.validationRegex && self.invalidMessage) || (self.validationRegex && !self.invalidMessage) ){
+        @throw [NSException exceptionWithName:NSInvalidArgumentException
+                                       reason:@"Both regex and invalid message properties must be set."
+                                     userInfo:nil];
+    }
+
```

Shall we also throw exception on an invalid `validationRegex`?